tests/QueryObjectTest.php

```diff
@@ -28,6 +28,11 @@ public function setUp(): void
         $this->queryObject = new SimpleQueryObject('simples');
     }
 
+    private function replaceWhitespace(string $string) {
```

Comment:
I would prefer if you replace the method with an assertion, something like assertStringsEqualWithoutWhitespaces and apply that assertion to all assertions in this class.